### DISCUSSION TITLE
Use of IP address rather than domain name in argocd-actions

### DISCUSSION THREAD

**Discussion by hyousif-pkglobal** [omegion/argocd-actions#7] [General]
Hello,

The sync action works well for an application that has been added already to ArgoCD. However, for a new repo that has never been connected to ArgoCD, the action fails and throws the following message:
'error: rpc error: code = NotFound desc = applications.argoproj.io "newrepo" not found'

It would be so helpful to add a step to connect to a new repo, and create a new app before syncing the application.

Thanks

hyousif

**Comment by omegion** [comment]
Hi,

It sounds a great idea, however, this action is all about basic operations on existing apps.
